src/main/resources/db/changesets/20181203_studytable.yaml

```diff
@@ -0,0 +1,109 @@
+databaseChangeLog:
+  - changeSet:
+      id: study_init
+      author: jhert
+      changes:
+        - createTable:
+            tableName: study
+            columns:
+              - column:
+                  name: id
```

Comment:
Is there a way to add description/documentation to this? Alternately, perhaps we make a document for the repository metadata schema. My preference is to keep documentation as close to the code as possible.

Reply:
We can add comments here with `#`. I like keeping the docs close to the code but migration files don't feel like the right place for documentation about the whole schema -- just information about the change being made by the migration.

Reply:
I added a story to the backlog to document the repository metadata schema.